dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -340,16 +340,15 @@ private DependencyGraphResult levelOrder(
                 resolutionException.getResult().getArtifactResults()) {
               if (artifactResult.getArtifact() == null) {
                 DependencyNode failedDependencyNode = artifactResult.getRequest().getDependencyNode();
-                ExceptionAndPath failure =
-                    ExceptionAndPath.create(parentNodes, failedDependencyNode, resolutionException);
-                artifactProblems.add(new UnresolvableArtifactProblem(failure.getPath()));
+                artifactProblems.add(
+                    new UnresolvableArtifactProblem(failedDependencyNode.getArtifact(), parentNodes));
```

Comment:
Memo: the parentNodes.push(dependencyNode) happens at line 327 (before this line).

Reply:
next PR